### ISSUE TITLE
[One Plus family] Stuck after unlock Bootloader waiting for device

### ISSUE THREAD

**Issue by freechelmi** [openandroidinstaller-dev/openandroidinstaller#115]
assisting people in Unlocking bootloaders is hard :-) 
Every manufacturer has a different GUI and instructions
Let's focus on oneplus on this issue

OAI guides well to the screen where the user have to choose to unlock with the volume and power button 

Then the next step is " fastboot reboot"  -> Not needed , one plus reboot itself and wipe until rebooting stock rom

Then OAI is stuck here , it wait to isse fastboot reboot command

![image](https://user-images.githubusercontent.com/446021/225862893-468846c5-1b7e-473e-85a8-0f9371d397b9.png)

Next step tells the user to reenable adb 

![image](https://user-images.githubusercontent.com/446021/225865516-379ccd0e-0f75-418e-9a04-d8be18fafdc8.png)

 we tell the user to shut down the phone , and manually reboot to fastboot ?

**Comment by freechelmi** [comment]
But If you go manually to fastboot , OAI will finally issue the "fastboot reboot " command and you go again to stock ROM :-)

**Comment by MagicLike** [comment]
Hmmm ok, there are (probably) multiple issues here...
First of all please tell me what OnePlus Model you have.

**Comment by tsterbak** [comment]
Hey @freechelmi, thank you for rising the issue!
Indeed, it should not boot after unlocking. This will be fixed in the next release. :)

**Comment by MagicLike** [comment]
> This will be fixed in the next release.

If it is ok, I will open a PR soon! (I would take the stuff out of my bacon config)

**Comment by MagicLike** [comment]
> > This will be fixed in the next release.
> 
> If it is ok, I will open a PR soon! (I would take the stuff out of my bacon config)

@tsterbak I just realised, that my PR #111 needs to be merged first as it also makes changes to all existing configs...

**Comment by MagicLike** [comment]
@tsterbak If it is fine, I will make the changes to all OnePlus configs now and open a PR soon...

**Comment by MagicLike** [comment]
> Great! Create a branch from the release branch if possible (#119).

Already done! ^^ #120

**Comment by MagicLike** [comment]
fixed in https://github.com/openandroidinstaller-dev/openandroidinstaller/pull/120/commits/b12f23969beb3a5e622c9a85507510ce8bada1f0

**Comment by tsterbak** [comment]
> This can now be closed - it will be implemented in the next release.

No hurry to close issues. For an issue like this where a lot of things can go wrong, I prefer to get the feedback from the reporting person :)

**Comment by tsterbak** [comment]
Hey @freechelmi, the latest release should have fixed this issue. If you want, you can give it a try :slightly_smiling_face:

**Comment by MagicLike** [comment]
@freechelmi Was the issue fixed with that update?

**Comment by tsterbak** [comment]
Any news on this?

**Comment by tsterbak** [comment]
> Sorry for my lack of answers , tested again with 0.5.1 and we jump correctly to , so fixed

Thanks a lot for testing again! Great that it works now! :)
